lab-judy/lib/storage.js

```diff
@@ -0,0 +1,50 @@
+'use strict';
+
+const Promise = require('bluebird');
+const createError = require('http-errors');
+const debug = require('debug')('note:storage');
```

Comment:
You should name your debug module after your model -- so it'll be like 'person:storage'.